agent/xds/delta.go

```diff
@@ -360,6 +364,11 @@ func (s *Server) processDelta(stream ADSDeltaStream, reqCh <-chan *envoy_discove
 
 			generator.Logger.Trace("Invoking all xDS resource handlers and sending changed data if there are any")
 
+			streamStartOnce.Do(func() {
+				diff := time.Since(streamStartTime)
+				metrics.AddSample([]string{"xds", "server", "streamStart"}, float32(diff/time.Millisecond))
```

Comment:
Is there a specific reason not to use [`MeasureSince`](https://pkg.go.dev/github.com/armon/go-metrics#MeasureSince) here?

Reply:
Oh, that's perfect. Fixed!